### PR TITLE
Module1:portfolio:mobile version 

### PR DESCRIPTION
**Pull request by hammaazarok**
*created a new branch with a different name as required by the previews reviewer
*add works-section
*add about me section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -34,6 +35,179 @@ <h1 class="header-text">I’m Hamma
...
+                    </ul>
+                </div>
+                <button type="button" class="work-btn btn">
+                        See Project
+                    </button>
+            </div>
+            <div class="work-card work-card2">
+                <img src="img/works/work2.png" alt="work1">
+
+                <h2 class="work-title">Multi-Post
```
Comment (tufoinnkuo10) [inline index.html:71]: - [ ] please use the correct figma font size and font weight for your card titles

File: index.html [hunk 2]
```diff
@@ -34,6 +35,179 @@ <h1 class="header-text">I’m Hamma
...
+                    </li>
+                    <li class="skill-item">
+                        <img src="./img/icons/js.png" alt="javascript">
+                        <p>javascript</p>
+                    </li>
+                    <li class="skill-item">
+                        <img src="./img/icons/html.png" alt="html">
+                        <p>HTML</p>
+                    </li>
+                    <li class="skill-item">
```
Comment (tufoinnkuo10) [inline index.html:190]: - [ ] please kindly reduce the space between the social media icons to be the same as those of figma, the direction icons should be the same also.
your design 
Screenshot (73)_LI
figma design 
Screenshot (72)_LI

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @hammaazarok 

Great job so far with your progress 😄 . There are some issues that you still need to work on to go to the next project but you are almost there!

`To Highlight`
✅ You used the correct Github flow
✅ Good commit messages
✅ Good READMe template
✅ Great pull request summary

`Required Changes` ♻️
Check the comments under the review

------

**Optional suggestions**
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by m05e5 — APPROVED** [review]
# Hi @hammaazarok , 

### Status: Approved ✔️
goodjob


:+1: You have done all the required changes asked by the previous reviewer. Well Done.

Your project is complete! 
Congratulations! 🎉 

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_m05e5_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**